node/out_response.js

```diff
@@ -218,10 +218,24 @@ TChannelOutResponse.prototype.sendError = function sendError(codeString, message
 TChannelOutResponse.prototype.emitFinish = function emitFinish() {
     var self = this;
 
-    if (!self.end) {
-        self.end = self.timers.now();
+    if (self.end) {
+        self.logger.warn('out response double emitFinish', {
+            end: self.end,
+            now: Date.now(),
+            serviceName: self.inreq.serviceName,
+            cn: self.inreq.headers.cn,
+            endpoint: String(self.inreq.arg1),
```

Comment:
that already exists as self.inreq.endpoint I found out, something @rf introduced towards tracing... it seems like a useful enough eager parsing to keep around and support for the eager req path, I plan to double down on it when killing arg1 streaming, thoughts?